### PR TITLE
Fix nullability quick-fixes with implicit receiver

### PR DESCRIPTION
**Pull request by nd**
#KT-17726

### CODE REVIEW

File: idea/testData/quickfix/nullables/unsafeInfixCall/unsafeInvokeWithImplicitReceiver.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// ERROR: Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type A?
+
+class A {
+    fun foo() {
+    }
+}
+
+fun A?.bar() {
+    <caret>foo()
+}
```
Comment (mglukhikh) [inline idea/testData/quickfix/nullables/unsafeInfixCall/unsafeInvokeWithImplicitReceiver.kt]: Why does this test have so strange name? I see no `invoke` here, either safe or unsafe

**Comment by mglukhikh** [comment]
Thank you. I'll try to take best from both this PR and https://github.com/JetBrains/kotlin/pull/1090

File: idea/testData/quickfix/nullables/unsafeInfixCall/unsafeInvokeWithImplicitReceiver.kt (hunk 1 above)
Comment (nd) [inline idea/testData/quickfix/nullables/unsafeInfixCall/unsafeInvokeWithImplicitReceiver.kt]: Right, it should be unsafeCallWithImplicitReceiver.kt..

**Comment by mglukhikh** [comment]
Merged into master

**Pull request closed without merging**